build-system/eslint-rules/forbidden-terms-config.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
       }
 
       const [nextComment] = context.getCommentsAfter(node);
-      const [nextCommentStart] = nextComment.range;
+      const [nextCommentStart] = nextComment?.range ?? [];
       if (
         nextComment &&
         text.substr(end, nextCommentStart - end).indexOf('\n') < 0
```

Comment:
Won't this break if `nextCommentStart` is undefined?

Reply:
You're right. Should be fine if we check `nextComment` around it.